### DISCUSSION TITLE
Android 14 Notes Role

### DISCUSSION THREAD

**Discussion by shamefulCake1** [gsantner/markor#2295] [Ideas] [answered]
Android 14 (sdk 34) supports the Notes Role for notetaking apps, which is a reified "notetaking feature".

https://x.com/MishaalRahman/status/1643281461171167235

I have not entirely understood what it is, but presumably there will be a way to either link, or directly forward places in apps and screenshots to those notetaking apps.
Google Keep plans to support this feature.

What about Markor?

**Comment by gsantner** [comment, marked as answer]
Not seems related to notes at first sight. Even if so, Markor is a Text Editor. Not a screenshot tool / image editor.

Hence, no.
